docs/source/api/recipes/recipes.ipynb

```diff
@@ -835,6 +835,73 @@
     "matching_ids = [k for k, v in H.degree().items() if v == d]\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.    print(len(e))
</pre><p>Can you execute the new cells (this one and the one above with the numerical statistic) so that the output is displayed? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>